lib/facter/simplib__numa.rb

```diff
@@ -0,0 +1,33 @@
+# _Description_
+#
+# Return a structured NUMA fact with possible nodes, offline nodes
+# and a memory mapping layout
+#
+Facter.add("simplib__numa") do
+  confine :kernel => 'Linux'
+  confine { File.exist? '/sys/devices/system/node' }
+
+  setcode do
+    result = {}
+
+    # read the 'possible' nodes file
```

Comment:
Nice!

Sorry that I didn't think about this earlier, but you may want to add `if File.exist?('<whatever>')` to these just so you don't throw a weird error.

I mean they *should* be there but :-|.

Reply:
Just added now :)